cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -972,6 +972,32 @@ def create_formatted_datetime_string_columns(df):
     datetime_format_dict = {
         "visit_datetime_string": "visit_datetime",
         "samples_taken_datetime_string": "samples_taken_datetime",
+        "household_digital_enrolment_invited_datetime_string": "household_digital_enrolment_invited_datetime",
```

Comment:
Hey @mat-may, if "_string" columns were missing for these it's an indicator that they are missing from the `cis_digital_datetime_map`. This map is used below to create the _string versions of all of the digital datetime fields. I'm not sure why we took a different approach for digital, but it ensure that the datetime map is complete. Please could you move any of these over if missing, and remove them from here?

Happy to chat in the morning if the relationship is not clear here

`date_of_birth` is also duplicated here and in the date variable list below, but should only appear in the date one.